backend/src/test/java/botobo/core/auth/AuthAcceptanceTest.java

```diff
@@ -38,11 +37,31 @@
 
         given(githubOauthManager.getUserInfoFromGithub(any())).willReturn(githubUserInfoResponse);
 
-        return RestAssured.given().log().all()
-                .contentType(MediaType.APPLICATION_JSON_VALUE)
-                .body(loginRequest)
-                .when().post("/api/login")
-                .then().log().all()
-                .extract();
+        return request()
+                .post("/api/login", loginRequest)
+                .build()
+                .extractableResponse();
+    }
+
+    @Test
+    @DisplayName("로그인을 한다 - 성공")
+    void login() {
+        // given
+        LoginRequest loginRequest = new LoginRequest("githubCode");
+        GithubUserInfoResponse githubUserInfoResponse = GithubUserInfoResponse.builder()
+                .userName("githubUser")
+                .githubId(1L)
+                .profileUrl("github.io")
+                .build();
+        given(githubOauthManager.getUserInfoFromGithub(any())).willReturn(githubUserInfoResponse);
+
+        // when
+        final HttpResponse response = request()
+                .post("/api/login", loginRequest)
+                .build();
+
+        // then
+        final TokenResponse tokenResponse = response.convertBody(TokenResponse.class);
```

Comment:
```suggestion
        final TokenResponse tokenResponse = 로그인되어_있음();
```
위에 이미 만들어진게 있다보니 이렇게만 해도 가능하더라고요ㅎㅎ;

Reply:
천재??